packages/aperf/aperf.0.1.1/opam

```diff
@@ -0,0 +1,16 @@
+opam-version: "1.2"
+maintainer: "Philip Dexter <philip.dexter@gmail.com>"
+authors: "Philip Dexter <philip.dexter@gmail.com>"
+homepage: "http://github.com/philipdexter/aperf"
+bug-reports: "http://github.com/philipdexter/aperf/issues"
+dev-repo: "http://git.phfilip.com/aperf"
```

Comment:
thanks!  This is not a valid version control url; needs to be:

```
dev-repo: "http://git.phfilip.com/aperf.git"
```

does that url actually work? :-)